test/root/predicates/predicateTestUtils.js

```diff
@@ -32,13 +32,15 @@ export function getAge(utxo) {
 
 // this is hack to generate a raw tx that is expected to be inflight
```

Comment:
you can generate rawTX using web3 why go with this hack?

Reply:
Yes, I got a little lazy while writing this one (since it involves instantiating a contract again, passing nonce etc and doesn't let me use truffle artifacts.) So thought of this quick hack 😅